docs/sts/cosign-dep.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cosign-sts
+  namespace: cosign
+spec:
+  selector:
+    matchLabels:
+      app: cosign-sts
+  template:
+    metadata:
+      labels:
+        app: cosign-sts
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65533
```

Comment:
Fixed user id's are not allowed with the `restricted-v2` SCC

Reply:
I removed the runAsUser